msdbook/tests/test_generalized_fish_game.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import numpy as np
+import matplotlib.pyplot as plt
+from msdbook.generalized_fish_game import (
+    inequality,
+    plot_uncertainty_relationship,
+    plot_solutions,
+    fish_game,
+    hrvSTR
+)
+
```

Comment:
should have 2 lines between imports and functions as well as 2 lines between functions according to [PEP 8](https://peps.python.org/pep-0008/#blank-lines)